fitbenchmarking/fitbenchmarking
# Issue #208: Add tests to run the example scripts

#### Description of Work

Fixes #208


#### Testing Instructions

1. Check Travis has run the additional tests
2. Check modified example scripts still run as standalone scripts
